src/histolab/tile.py

```diff
@@ -29,6 +29,30 @@ def __init__(self, image: PIL.Image.Image, coords: CoordinatePair, level: int =
         self._coords = coords
         self._level = level
 
+    def apply_filters(
+        self,
+        filters: Union[
+            Callable[[PIL.Image.Image], Union[PIL.Image.Image, np.ndarray]], imf.Compose
+        ],
+    ) -> "Tile":
+        """Apply a filter or composition of filters on a tile.
+
+        Parameters
+        ----------
+        filters : Union[ Callable[[PIL.Image.Image], Union[PIL.Image.Image, np.ndarray]]
+                , imf.Compose]
+            Filter or composition of filters to be applied
+
+        Returns
+        -------
+        Tile
+            Tile with the filters applied
+        """
+        filtered_image = filters(self.image)
+        if isinstance(filtered_image, np.ndarray):
+            filtered_image = PIL.Image.fromarray(filtered_image)
```

Comment:
in which condition this could be a `ndarray`?
I mean, if self.image is a known property and we apply a filter composition to it, the result type should be of 1 type

me == confused

Reply:
It depends on the transformations: if at the end we have an OtsuThreshold then the result will be a numpy array

Reply:
got it